recipes/squash/build.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -ex
+
+EXTRA_CMAKE_ARGS=""
+if [[ `uname` == 'Darwin' ]]; then
+  export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+  # We override most toolchain variables to modify the min version for OSX.
+  export MACOSX_VERSION_MIN="10.7"
```

Comment:
I am not sure these are the recommended ones from the `toolchain`. Please double check to ensure consistency.